server/mlabns/util/util.py

```diff
@@ -1,12 +1,15 @@
 import json
+import os
 
 import jinja2
 
 from mlabns.util import message
 
 def _get_jinja_environment():
+    current_dir = os.path.dirname(__file__)
```

Comment:
I instinctively think this should be os.getcwd() ?  The magic __file__ and __name__ variables make me very nervous...